media-proxy/Dockerfile

```
@@ -182,3 +182,52 @@ SHELL ["/bin/bash", "-c"]
 ENTRYPOINT ["/usr/local/bin/media_proxy"]
 
 HEALTHCHECK --interval=30s --timeout=5s CMD ps aux | grep "media_proxy" || exit 1
+
+
+FROM ${IMAGE_CACHE_REGISTRY}/${IMAGE_NAME} as mesh-agent
+
+LABEL maintainer="milosz.linkiewicz@intel.com"
+LABEL org.opencontainers.image.title="Media Communications Mesh Media Proxy Agent"
+LABEL org.opencontainers.image.description="Media Communications Mesh Media Proxy Agent application. Ubuntu 22.04 Docker Container Release Image"
```

It sounds very bad with two meshes next to each other, perhaps we can come up with something even better?
```suggestion
LABEL org.opencontainers.image.title="Media Communications Mesh Mesh Agent"
LABEL org.opencontainers.image.description="Media Communications Mesh Mesh Agent application. Ubuntu 22.04 Docker Container Release Image"
```

---

@Magda-Pytel Consider the following names
* Agent -> Media Communications Mesh Agent
* Proxy -> Media Communications Mesh Media Proxy

There is no such SW component like `Media Proxy Agent` in MCM.

---

I think Konstantin's version is much clearer. Mine came from confusion.